contracts/governance/Vesting/VestingStorage.sol

```diff
@@ -24,7 +24,7 @@ contract VestingStorage is Ownable {
     address public tokenOwner;
 
     /// @notice Fee sharing Proxy.
-    IFeeSharingProxy public feeSharingProxy;
+    IFeeSharingCollector public feeSharingCollectorProxy;
```

Comment:
same here. just calling it feeSharingCollector would be easier

Reply:
updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/482/commits/61d88fbfe3ceb4b74fa8b612479c397faca5c290)